lib/xslt/iso2geoBL.xsl

```diff
@@ -10,121 +25,126 @@
+        <xsl:when test="contains(gmd:organisationName/gco:CharacterString, 'Scholars&apos;)">
+          <xsl:text>UVa</xsl:text>
         </xsl:when>
-      </xsl:choose>
-    </xsl:variable>
+        
+        </xsl:choose>
+    </xsl:for-each>
+  </xsl:variable>
+  
+<!-- bounding box -->
+<xsl:variable name="upperCorner">
+  <xsl:value-of select="number(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:geographicElement/gmd:EX_GeographicBoundingBox/gmd:eastBoundLongitude/gco:Decimal)"/>
+  <xsl:text> </xsl:text>
+  <xsl:value-of select="number(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:geographicElement/gmd:EX_GeographicBoundingBox/gmd:northBoundLatitude/gco:Decimal)"/>
+</xsl:variable>
+  
+  <xsl:variable name="lowerCorner">
 
-    <xsl:variable name="uuid">
-      <xsl:choose>
-        <xsl:when test="gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/gmd:CI_Citation/gmd:identifier/gmd:MD_Identifier/gmd:code">
-          <xsl:value-of select="gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/gmd:CI_Citation/gmd:identifier/gmd:MD_Identifier/gmd:code"/>
-        </xsl:when>
-        <xsl:when test="gmd:MD_Metadata/gmd:distributionInfo/gmd:MD_Distribution/gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource/gmd:linkage/gmd:URL">
-          <xsl:value-of select="gmd:MD_Metadata/gmd:distributionInfo/gmd:MD_Distribution/gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource/gmd:linkage/gmd:URL"/>
-        </xsl:when>
-      </xsl:choose>
-    </xsl:variable>
+    <xsl:value-of select="number(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:geographicElement/gmd:EX_GeographicBoundingBox/gmd:westBoundLongitude/gco:Decimal)"/>
+      <xsl:text> </xsl:text>
+    <xsl:value-of select="number(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:geographicElement/gmd:EX_GeographicBoundingBox/gmd:southBoundLatitude/gco:Decimal)"/>
 
-    <xsl:variable name="identifier">
-      <xsl:choose>
-        <!-- for Stanford URIs -->
-        <xsl:when test="contains($uuid, 'purl')">
-          <xsl:value-of select="substring($uuid, string-length($uuid)-10)"/>
-        </xsl:when>
-        <!-- all others -->
-        <xsl:otherwise>
-          <xsl:value-of select="$uuid"/>
-        </xsl:otherwise>
-         </xsl:choose>
-    </xsl:variable>
+  </xsl:variable>
+  <xsl:variable name="x2" select="substring-before($upperCorner/text(), ' ')"/><!-- E -->
+  <xsl:variable name="x1" select="substring-before($lowerCorner/text(), ' ')"/><!-- W -->
+  <xsl:variable name="y2" select="substring-after($upperCorner/text(), ' ')"/><!-- N -->
+  <xsl:variable name="y1" select="substring-after($lowerCorner/text(), ' ')"/><!-- S -->
```

Comment:
Similar issue as listed above in fgdc2geoBl. This just doesn't work with Nokogiri and fails tests.